src/main/java/at/hannibal2/skyhanni/data/MiningAPI.kt

```diff
@@ -57,11 +52,13 @@ object MiningAPI {
             updateCold(0)
             lastColdReset = SimpleTimeMark.now()
         }
-        coldResetDeath.matchMatcher(event.message) {
-            if (group("name") == LorenzUtils.getPlayerName()) {
-                updateCold(0)
-                lastColdReset = SimpleTimeMark.now()
-            }
+    }
+
+    @SubscribeEvent
+    fun onPlayerDeath(event: PlayerDeathEvent) {
```

Comment:
we could early return if the player name is not the user but not necessary

Reply:
early return is not needed and rather confusing when its the only check in the function imo

Reply:
also this is no precondition, but THE condition